modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -47,7 +50,9 @@ ifeval::["{context}" == "about-openshift-sdn"]
 
 |IPsec encryption|Supported
 
-|IPv6|Supported ^[2]^ ^[3]^
+|IPv6 single-stack networking|Not supported|Supported ^[2]^ ^[3]^
```

Comment:
If you will accept [cybertron](https://github.com/cybertron)'s comment below, this will be ^[3]